java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/symbol/SymbolProvider.java

```diff
@@ -193,6 +193,9 @@ default boolean queryQualificationMatches(String query, ICompilationUnit unit, L
         } catch(Exception e) {
             logInfo("unable to make unit consistant, will still try as could be class file in a jar" + e);
         }
+        // should consider parameter here
+        // e.g. java.nio.file.Paths.get(String)/java.nio.file.Paths.get(*)  -> java.nio.file.Paths.get
+        query = query.replaceAll("\\([^\\)]*\\)", "");
```

Comment:
This is good. There is actually a [PR](https://github.com/konveyor/rulesets/pull/262) I have made where these are all fixed manually, but it still needs to be merged.

Reply:
The parameters were removed because they are not needed in the `queryQualificationMatches` function. However, under the current function logic, the parameters may lead to some unexpected errors. e.g. `java.nio.file.Paths.get(*)` will become `java.nio.file.Paths.get(.*)`. You are right, my previous version didn’t handle types, so I updated it to simply remove everything within parentheses now.